### PR TITLE
Update AboutUs and PPP

### PR DESCRIPTION
**Pull request by yuqitanyq**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -24,26 +24,26 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 * Responsibilities: UI
 
-### Johnny Doe
+### Tan Yu Qi
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/yuqitanyq.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/yuqitanyq)] [[portfolio](team/yuqitanyq.md)]
```
Comment (Halpfrog) [inline docs/AboutUs.md:36]: Take care to make sure that there won't be refactoring issues when we change links.

**Review by Halpfrog — CHANGES_REQUESTED** [review]
Change the links back to accessible ones to make sure they don't all links to the same profile.

File: docs/team/yuqitanyq.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* **Documentation**:
+  * User Guide:
+  * Developer Guide:
+  
+
+* **Community**:
+  
+
+* **Tools**:
+
```
Comment (yl-ang) [inline docs/team/yuqitanyq.md:34]: LGTM!

**Review by yl-ang — COMMENTED** [review]
LGTM!

**Pull request merged**